### PR TITLE
Change seq

### PR DESCRIPTION
**Pull request by andrewbaxter439**
Crucially - the Telegram message update only works for me or can be replicated by anyone who wants to set up their own account. Removed as redundant.

Changed sequencing to end of parallel call (which isn't yet tested). Big thing missing here is that this is hard-coded to do 50 parallel runs with the option of how many runs within each. To change number of parallels this would require a re-calculation to pass that many seeds to parallel.

### CODE REVIEW

File: do_full_run.sh [hunk 1]
```diff
@@ -134,9 +135,4 @@ rm -r ./output/202*/database ./output/202*/input
 # WARNING: I/O heavy, saturates SATA3, requires extra temporary space
 sh ./remove_trailing_commas.sh
 
-# Text myself that it's all done
-{ # try
-  curl "https://api.telegram.org/bot${Notify_bot_key}/sendMessage?text=Done%20copying%20all%20files&chat_id=${telegram_chatid}" &&
-} || { # catch
-  echo "Failed to send a confirmation, logging success anyway."
-}
+echo "Successfully completed runs"
```
Comment (vkhodygo) [inline do_full_run.sh:138]: We could probably set up a mail notification, but that's not worth the time at the moment.

File: do_full_run.sh [hunk 2]
```diff
@@ -120,12 +120,13 @@ then
...
-                                                                                             -e $END_YEAR \
-                                                                                             -g $GUI \
-                                                                                             -f
+parallel java -cp simpaths.jar simpaths.experiment.SimPathsMultiRun -r {} -n $BATCH_SIZE \
+                                                                          -p $POPULATION_SIZE \
+                                                                          -s $START_YEAR \
+                                                                          -e $END_YEAR \
+                                                                          -g $GUI \
+                                                                          -f \
+																		  ::: {100..5000..100}
```
Comment (vkhodygo) [inline do_full_run.sh]: Try a dry run on the machine and see how it goes

**Comment by vkhodygo** [comment]
> Big thing missing here is that this is hard-coded to do 50 parallel runs with the option of how many runs within each. To change number of parallels this would require a re-calculation to pass that many seeds to parallel.

Do we actually need this at the moment? I intentionally didn't touch this part as it works just fine.

File: do_full_run.sh (hunk 1 above)
Comment (andrewbaxter439) [reply do_full_run.sh:138]: I'd agree. At some point I'd like to look at setting up a [ntfy.sh](https://ntfy.sh/) server for pushing/subscribing to long processes that we do in the Unit but not quite a priority!

**Comment by andrewbaxter439** [comment]
> > Big thing missing here is that this is hard-coded to do 50 parallel runs with the option of how many runs within each. To change number of parallels this would require a re-calculation to pass that many seeds to parallel.
> 
> Do we actually need this at the moment? I intentionally didn't touch this part as it works just fine.

Only problem it may cause in the medium term is runs of more than 150,000 persons - 50 parallel runs of 1,000,000 plus may take too much RAM. We can always revisit and manually edit code when we come to this?

File: do_full_run.sh (hunk 2 above)
Comment (vkhodygo) [reply do_full_run.sh]: Any news?

**Comment by andrewbaxter439** [comment]
This is still on my radar - so far failing to get the main branch working then will test this version

**Comment by andrewbaxter439** [comment]
@vkhodygo These changes now work and can be merged

**Review by vkhodygo — APPROVED** [review]

**Pull request merged**